driver/driver.cc

```diff
@@ -148,6 +148,14 @@ static llvm::cl::opt<int> IpuNum(
     llvm::cl::desc("Num of ipus, should consistent with subgraph num"),
     llvm::cl::init(1), llvm::cl::cat(HaloOptCat));
 
+static llvm::cl::opt<bool> EnableEngineCache(
+    "enable-engine-cache", llvm::cl::desc("Enable popart engine cache"),
```

Comment:
We don't need to modify halo. We should use odla_* APIs from user space.

Reply:
ok，i found `odla_StoreExecutable/odla_LoadExecutable` is already defined, will implemented in odla_popart.cc, EnableEngineCache option reserved for call halo binary scenario